lnrpc/neutrinorpc/neutrino.proto

```diff
@@ -207,4 +212,14 @@ message GetCFilterRequest {
 message GetCFilterResponse {
     // GCS filter.
     bytes filter = 1;
-}
\ No newline at end of file
+}
+
+message GetBlockHashRequest {
+    // The block height or index.
```

Comment:
or index? Did you mean the height index?

Reply:
Changed it to height index.